prow/istio-integ-local-tests.sh

```diff
@@ -31,8 +31,6 @@ setup_and_export_git_sha
 
 cd "${ROOT}"
 
-# Unit tests are run against a local apiserver and etcd.
```

Comment:
why remove this comment?

Reply:
this was copied from the unit test script and does not apply here.